psc-bundle/README.md

```diff
@@ -4,14 +4,15 @@ A dead code elimination tool for PureScript-style CommonJS modules. This can be
 
 ## Usage
 
-    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG]
-    
+    psc-bundle FILE (-m|--module ARG) [--main ARG] [--namespace ARG] [--optimize uncurry]
+
 Options:
 
 - The input .js file(s)
 - Entry point module name(s) are specified with `-m` or `--module`. All code which is not a transitive dependency of an entry point module will be removed.
 - The main module is (optionally) specified using `--main`. If specified, this will generate code to run the main method in the specified module.
 - The browser namespace defaults to `PS`, and can be overridden with `--namespace`.
+- The optimization options are specified with `-p` or `--optimize`.  Currently the only option is `uncurry` or `u`, which performs an uncurry optimization
```

Comment:
Also, `psc` has a separate command line flag (not option, i.e. no argument) for each optimization it supports. Unless there's a good reason not to, I think it would probably be good to have `psc-bundle` be consistent.